doc/src/performance_tuning.rst

```diff
@@ -175,7 +207,16 @@ prerequisites must be satisfied:
 
 After completing this process it is highly recommended to verify
 everything is working by trying the
-`h5py parallel hdf5 example <https://docs.h5py.org/en/stable/mpi.html#using-parallel-hdf5-from-h5py>`_.
+`h5py parallel HDF5 example <https://docs.h5py.org/en/stable/mpi.html#using-parallel-hdf5-from-h5py>`_.
+
+Plugins
+=======
+
+A common source of performance issues is running plugins too
+frequently.  Given the time steps taken by PyFR are typically much
+smaller than those associated with the underlying physics there is
+seldom any benefit to running integration and/or time average
+accumulation plugins more frequently than once every 50 steps.
```

Comment:
Maybe also add a comment about the nan check plugin having significant overhead in the current implementation as well. It can be a silent killer to performance.